### PR TITLE
add kvm=off for qemu

### PR DESCRIPTION
**Pull request by Jimmy-Xu**
Solve the problem of loading drivers for some nvidia graphics cards.

Fixes: #2546

> Just for test

### CODE REVIEW

**Comment by wenlxie** [comment]
Can we use the configuration to disable this but not make it as default? This will impact the performance?

**Comment by chestack** [comment]
came into the same problem, this patch worked for me.

File: virtcontainers/qemu_amd64.go [hunk 1]
```diff
@@ -129,6 +129,7 @@ func (q *qemuAmd64) bridges(number uint32) {
 
 func (q *qemuAmd64) cpuModel() string {
 	cpuModel := defaultCPUModel
+	cpuModel += ",kvm=off"
```
Comment (jodh-intel) [inline virtcontainers/qemu_amd64.go:132]: If this is really the solution for Nvidia GPUs, couldn't we just document to add `kvm=off` to `cpu_features=` in `configuration.toml`?

Aside: If the number of GPU-specific settings in `configuration.toml` grows, we might want to think about adding an actual GPU config option which if set automatically sets these "ancillary" config options to make life easier for GPU users.

Comment (devimc) [reply virtcontainers/qemu_amd64.go:132]: :+1:  please don't hard code this , instead you can document it as a GPU-specific setting for some specific GPUs

**Comment by fidencio** [comment]
As this PR is marked as `do-not-merge`, we're assuming it won't make for the final 1.13.0 release, thus we're closing it.
In case you feel this has been wrongly closed, please, feel free to re-open it.

**Pull request closed without merging**